### PR TITLE
tectonic: update to 0.4.0.

### PR DESCRIPTION
**Pull request by petrvelicka**

### CODE REVIEW

**Comment by Piraty** [comment]
i686 fails

File: srcpkgs/tectonic/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'tectonic'
 pkgname=tectonic
-version=0.3.3
+version=0.4.0
```
Comment (ericonr) [inline srcpkgs/tectonic/template:3]: 0.4.1 is out

**Review by ericonr — COMMENTED** [review]
The 32-bit failure might just be a lack of testing 32-bit on their side. Will open an issue.

https://github.com/tectonic-typesetting/tectonic/issues/749

**Comment by Piraty** [comment]
ping @petrvelicka

**Pull request closed without merging**